generic-visually-hidden/visually-hidden.css

```diff
@@ -0,0 +1,10 @@
+[visually-hidden] {
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  height: 1px;
+  width: 1px;
```

Comment:
Authors can make interactive elements `visually-hidden`, did you consider https://github.com/thepassle/generic-components/issues/7#issuecomment-635402275?

> ```css
>  /* give it non-zero size, VoiceOver on Safari requires at least 2 pixels
>      before allowing buttons to be activated. */
>   width: 4px !important;
>   height: 4px !important;
> ```

Reply:
🤔 That comment somehow slipped my attention, sorry about that. Will revisit and update 🙂

Reply:
> Authors can make interactive elements visually-hidden

I should add that, I believe `pointer-events: none` should be added to the snippet, because otherwise there will be a 4x4 clickable box. Either AMP didn't care because it's highly unlikely you'll actually hover it, or they did not consider it. I should  probably open an issue upstream.

Reply:
AMP [uses `pointer-events: none`](https://github.com/caroqliu/amphtml/blob/96752552de707611d88d9c5524d81a0f20db10e4/extensions/amp-carousel/0.1/amp-carousel.css#L96-L99) in their `amp-carousel` component in which they visually hide interactive elements. No need to upstream I suppose.

Reply:
ty for the update :)